README.md

```diff
@@ -294,15 +294,15 @@ This command is not available through the `adb` command. It lists files in a fol
 9. Device sends OKAY
 10. We send WRTE to device
 11. We send RECV to device
-12. Device sends a series of "dents" (directory entries), across an arbritrary number of packets of abritrary length - sometimes the packets are only 1 byte long. It will send OKAY when the next packet is ready. We keep returning to 10. (send WRTE, RECV, read DENT) until we receive DONE. "dents" can be split across several packets, but are the following structure: 
+12. Device sends a series of "dents" (directory entries), across an arbitrary number of packets of arbitrary length - sometimes the packets are only 1 byte long. It will send OKAY when the next packet is ready. We keep returning to 10. (send WRTE, RECV, read DENT) until we receive DONE. "dents" can be split across several packets, but are the following structure: 
     1. A four-byte response id DENT (or DONE, if there are no more files to list. DONE is sent immediately if the remote path does not exist or if it is not a folder)
     2. A four-byte integer representing file mode - first 9 bits of this mode represent the file permissions, as with chmod mode. Bits 14 to 16 seem to represent the file type, one of `0b100` (file), `0b010` (directory), `0b101` (symlink)
     3. A four-byte integer representing file size.
     4. A four-byte integer representing last modified time in seconds since Unix Epoch.
     5. A four-byte integer representing file name length.
     6. A utf-8 string representing the file name.
     7. Back to 1.
-18. Device sends DONE. DONE can appear in the start, middle or end of a packet. Sometimes the rest of the packet after DONE is padded with 0s.
+18. Device sends DONE. DONE can appear at the start, middle or end of a packet. Sometimes the rest of the packet after DONE is padded with 0s.
```

Comment:
Maybe the second sentence should actually be reworded, something like `DONE can appear anywhere within the packet`?

Reply:
Please review the sentence. Thanks